graylog2-server/src/main/java/org/graylog2/plugin/indexer/searches/timeranges/AbsoluteRange.java

```diff
@@ -29,6 +30,7 @@
 import java.util.Map;
 
 @AutoValue
+@WithBeanGetter
```

Comment:
This class, `KeywordRange` and `RelativeRange` all implement their own `#getFrom()` and `#getTo()` due to the abstract methods in `TimeRange`.

Maybe we should remove the annotation from those to avoid confusion in the future if any of those hand written implementations should change.